tests/check/check_h26xsigned_sign.c

```diff
@@ -579,6 +581,132 @@ START_TEST(undefined_nalu_in_sequence)
 }
 END_TEST
 
+/* Test description
+ * Verify that after 2 completed SEIs created and will be emitted in correct order
+ * The operation is as follows:
+ * 1. Setup a signed_video_t session
+ * 2. Add 2 I NALU for signing that will trigger 2 SEI NALU
+ * 3. Get the SEI NALUs to prepend
+ * 4. Check that SEI NALUs were emitted in correct order
+ */
+START_TEST(two_completed_seis_pending)
+{
+  // By construction, run the test for SV_AUTHENTICITY_LEVEL_FRAME only.
+  if (settings[_i].auth_level != SV_AUTHENTICITY_LEVEL_FRAME) return;
+
+  SignedVideoCodec codec = settings[_i].codec;
+  SignedVideoReturnCode sv_rc;
+  size_t sei_size_1 = 0;
+  size_t sei_size_2 = 0;
+  size_t sei_size_3 = 0;
+  signed_video_t *sv = signed_video_create(codec);
+  sv->sv_test_on = true;
```

Comment:
Add a comment that you are bypassing code for checking if there are available SEIs